src/evo/assetlocktx.cpp

```diff
@@ -118,9 +118,8 @@ bool CAssetUnlockPayload::VerifySig(const uint256& msgHash, const CBlockIndex* p
 
     // We check at most 2 quorums
     const auto quorums = llmq::quorumManager->ScanQuorums(llmqType, pindexTip, 2);
-    bool isActive = std::any_of(quorums.begin(), quorums.end(), [&](const auto &q) { return q->qc->quorumHash == quorumHash; });
 
-    if (!isActive) {
+    if (bool isActive = std::any_of(quorums.begin(), quorums.end(), [&](const auto &q) { return q->qc->quorumHash == quorumHash; }); !isActive) {
```

Comment:
I think the reason it was written in 2 lines is because otherwise the condition inside `if` is too long.
If you still want it in one line you could shrink it to
```suggestion
    if (!std::any_of(quorums.begin(), quorums.end(), [&](const auto &q) { return q->qc->quorumHash == quorumHash; })) {
```
I think.

Reply:
I thought about both these simplifications but I think the named variable acts as a comment to improve readability.